.changeset/flat-emus-remain.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudfour/patterns': minor
```

Comment:
I believe that with @spaceninja's addition this is now a `major` change (as it will break spacing if templates don't transition to using `heading`).